backend/fms_core/viewsets/experiment_run.py

```diff
@@ -80,7 +80,7 @@ def set_experiment_run_lane_validation_status(self, _request):
         run_name = _request.data.get("run_name", None)
         lane = _request.data.get("lane", None)
         validation_status = _request.data.get("validation_status", None)
-        validation_status = int(validation_status) if validation_status else None
+        validation_status = int(validation_status) if validation_status is not None else None
```

Comment:
In the frontend I was expecting to get AVAILABLE, PASSED, or FAILED. Instead I was getting null. I thought it was a bug, so I fixed it. :) For the frontend, it is much clearer if we use AVAILABLE, so that we can always expect some value in this field. With the UX change I just made, users will no longer be able to reset the status to AVAILABLE - only staff will be able to clear the validation status, so we shouldn't have to worry about the problem you are describing here.